### PR TITLE
Add FragDenStaat

### PR DESCRIPTION
**Pull request by juliuste**
This PR adds an entry for the German community-maintained FOIA (freedom of information act) portal [FragDenStaat](https://fragdenstaat.de). I hope it fits the criteria, since its current rank is within the top 200k domains on Similarweb.

I was not 100% sure about the following 2 attributes, please let me know if I should change anything:
- despite being a service focused on Germany, the website doesn't have any location restrictions, I therefore didn't add a `regions` property
- the website is clearly concerned with government topics, but at the same time it is community-maintained, so the usage of the `government` keyword might be debatable

Thank you very much for maintaining this project!

### CODE REVIEW

File: img/f/fragdenstaat.de.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg width="456" height="399" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M76.4729 171.01v12.414l43.3501 25.812V171.01H76.4729ZM156.768 231.207l72.513 43.054 70.837-42.857v-44.335l-72.315 42.857-71.035-42.759v44.04ZM337.064 171.01v38.029l43.35-26.207V171.01h-43.35Z" fill="#296DFF"/><path d="M428 171.01v-37.931h-25.813L229.773 25 52.9261 133.079H28v37.931h11.5271v164.039H28v37.931h400v-37.931h-10.64V171.01H428Zm-127.882 0v164.039h-143.35V171.01h143.35ZM227.31 69.335l108.867 63.744H123.665L227.31 69.335ZM76.4729 171.01h43.3501v164.039H76.4729V171.01ZM380.414 335.049h-43.35V171.01h43.35v164.039Z" fill="#C9D2DC"/></svg>
```
Comment (phallobst) [inline img/f/fragdenstaat.de.svg:1]: Slightly optimized, without padding:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 400 348"><path fill="#296dff" d="M48.5 146v12.4c50.9 30.4 101.8 60.6 152.8 90.9l151.1-91.5V146h-53.6l-99 59-98-59z"/><path fill="#c9d2dc" d="M400 146v-38h-25.8L201.8 0 24.9 108H0v38h11.5v164H0v38h400v-38h-10.6V146Zm-127.9 0v164H128.8V146ZM199.3 44.3l108.9 63.8H95.7ZM48.5 146h43.3v164H48.5Zm304 164H309V146h43.3z"/></svg>
```

File: entries/f/fragdenstaat.de.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "FragDenStaat": {
+    "domain": "fragdenstaat.de",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://fragdenstaat.de/hilfe/ihr-konto-verwalten/zwei-faktor-login/",
+    "keywords": [
+      "government"
+    ]
```
Comment (phallobst) [inline entries/f/fragdenstaat.de.json:14]: There may not be a restriction for the accounts, but given the target audience we should probably use a region.
```suggestion
    ],
    "regions": [
      "de"
    ]
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/f/fragdenstaat.de.svg (hunk 1 above)
Comment (Carlgo11) [inline img/f/fragdenstaat.de.svg]: The padding is unnecessary. Here's a version without any padding/margin around the icon: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 348"><path fill="#296dff" d="M48.5 146v12.4L200 249.3l152.4-91.5V146h-43.3L200 205 91.8 146Z"/><path fill="#c9d2dc" d="M400 146v-38h-25.8L200 0 25 108H0v38h11.5v164H0v38h400v-38h-10.6V146Zm-127.9 0v164H128.8V146ZM200 44.3l108.2 63.8H95.7ZM48.5 146h43.3v164H48.5Zm304 164H309V146h43.3z"/></svg>
```

File: entries/f/fragdenstaat.de.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "FragDenStaat": {
+    "domain": "fragdenstaat.de",
+    "tfa": [
+      "totp",
+      "u2f"
```
Comment (Carlgo11) [inline entries/f/fragdenstaat.de.json:6]: Is this implemented as a passwordless authentication or a second factor?

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @juliuste!
I would very much like the de region tag to be included as other regions won't have much use of the site :wink:

Whether or not it fits in the Government category I'll leave to the German maintainers to decide.

**Comment by kmpoppe** [comment]
> Whether or not it fits in the Government category I'll leave to the German maintainers to decide.

If we see `government` as government-"related", it might be correct, but afaik all existing entries currently are government-"run". I could live with it, but I'd also accept `legal`...

K

File: entries/f/fragdenstaat.de.json (hunk 3 above)
Comment (juliuste) [reply entries/f/fragdenstaat.de.json:6]: afaik 2nd factor "only"

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**